hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/util/S3StorageType.java

```diff
@@ -48,8 +50,17 @@ public ReplicationType getType() {
     return type;
   }
 
-  public static S3StorageType getDefault() {
-    return STANDARD;
+  public static S3StorageType getDefault(ConfigurationSource config) {
```

Comment:
Can we add JavaDoc for this.

And also what is the reason if config has wrong value, falling back to STANDARD?

Reply:
@bharatviswa504 sure, I'll add more descriptions.
Yes, for now I just try to fallback to STANDARD.